src/test_array_builder.cc

```diff
@@ -51,6 +51,12 @@ std::vector<tiledb_datatype_t> dimension_dense_datatypes = {
     TILEDB_UINT32,
     TILEDB_INT64,
     TILEDB_UINT64,
+    /*TILEDB_STRING_ASCII,
```

Comment:
I know that this source file already contains similar block-comments, but we typically do not want to check in commented-out code (or dead) code. Please remove these lines.